### ISSUE TITLE
Optimal Transport Based Refinement of Physics-Informed Neural Networks. (arXiv:2105.12307v1 [cs.CE])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#7325]
Optimal Transport Based Refinement of Physics-Informed Neural Networks. (arXiv:2105.12307v1 [cs.CE])<br>
https://ift.tt/3upgoME<br>
<p>In this paper, we propose a refinement strategy to the well-known Physics-Informed Neural Networks (PINNs) for solving partial differential equations (PDEs) based on the concept of Optimal Transport (OT).</p>
<p>Conventional black-box PINNs solvers have been found to suffer from a host of issues: spectral bias in fully-connected architectures, unstable gradient pathologies, as well as difficulties with convergence and accuracy.</p>
<p>Current network training strategies are agnostic to dimension sizes and rely on the availability of powerful computing resources to optimize through a large number of collocation points.</p>
<p>This is particularly challenging when studying stochastic dynamical systems with the Fokker-Planck-Kolmogorov Equation (FPKE), a second-order PDE which is typically solved in high-dimensional state space.</p>
<p>While we focus exclusively on the stationary form of the FPKE, positivity and normalization constraints on its solution make it all the more unfavorable to solve directly using standard PINNs approaches.</p>
<p>To mitigate the above challenges, we present a novel training strategy for solving the FPKE using OT-based sampling to supplement the existing PINNs framework.</p>
<p>It is an iterative approach that induces a network trained on a small dataset to add samples to its training dataset from regions where it nominally makes the most error.</p>
<p>The new samples are found by solving a linear programming problem at every iteration.</p>
<p>The paper is complemented by an experimental evaluation of the proposed method showing its applicability on a variety of stochastic systems with nonlinear dynamics.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
